CHANGELOG.md

```diff
@@ -5,6 +5,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.11.0 - 2021-08-27
+
+### Added
+
+- Add official compatibility with Tartiflette 1.4.x. (Pull #153)
+
+### Changed
+
+- Update dependency on Starlette to `0.16.*`. (Pull #154)
+- Refactor internal subscription streams (Pull #156)
```

Comment:
Don't think we should include this in the changelog, as it's a purely internal, non-user-impacting change?

```suggestion
```